docs/contributor-guide/getting started/development-environment.md

```diff
@@ -8,23 +8,15 @@ environment that will let you contribute back to the project.
 3. [Add the upstream source](#step-3)
 4. [Run the tests](#step-4)
 
-<!-- markdownlint-disable MD033 -->
-
-## Step 1: Install Node.js <a name="step-1"></a>
```

Comment:
Is there a reason why the `<a name="step-n"></a>` part was removed here and below?

Reply:
Yes, we are adding the links automatically when processing the markdown and that was generating 2 links.

Reply:
👍